components/button/Button.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Button } from "./Button";
+
+it('should render a button with no variant defined', () => {
```

Comment:
Later we'll need to wrap this up into a Describe. Then, we can test each button variant.

Reply:
I just looked it up and that makes sense. Shall I add that wrapper now so it's there and we can build on it later?

Reply:
Later.